skilld-labs/zen
# Gulp clone taks instead drush zen command. Cleanup unsed zen theme files

How to check:
* install fresh drupal
* `cd themes`
* `drush en components -y`
* `git clone git@github.com:skilld-labs/zen.git && cd zen && git checkout gulp-clone-task`
* `cd zen`
* `docker run --name zen-container --rm -v $(pwd):/work -v $(pwd)/../gulpclone:/gulpclone -it skilldlabs/frontend:zen sh -c 'npm i gulp-regex-rename yargs && gulp --theme_name gulpclone'
`
* `drush en gulpclone -y`
* `drush config-set system.theme default gulpclone -y` 
* check what theme works correctly and has not errors in dblog reports
